docker/debug/Dockerfile.debug

```diff
@@ -14,6 +14,12 @@ RUN go get -u golang.org/x/tools/cmd/stringer
 
 RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
 
+ENV GO111MODULE=on
+RUN mkdir /root/networkservicemesh
+ADD ["go.mod","/root/networkservicemesh"]
+WORKDIR /root/networkservicemesh/
+RUN go mod download
+
```

Comment:
This change probably should belong to another PR.

Reply:
ok